src/cryptography/hazmat/bindings/openssl/binding.py

```diff
@@ -135,7 +135,7 @@ def _ensure_ffi_initialized(cls):
                 cls._register_osrandom_engine()
 
     @classmethod
-    def init_static_locks(cls):
+    def _init_static_locks(cls):
```

Comment:
Hm, that's an interesting change. I know of at least one project which will break but I'm happy to remove the call if it makes no sense:

https://github.com/aiortc/aiortc/blob/28cf22c2a82136784cc8ece6ca82143afdba5c86/aiortc/rtcdtlstransport.py#L40

Reply:
`init_static_locks()` is called at module scope, so there's no way to import `Binding` without it already having been called, so yeah, it doesn't really make sense